zap/src/main/java/org/parosproxy/paros/extension/option/OptionsViewPanel.java

```diff
@@ -94,6 +95,10 @@
     // ISO Standards compliant format
     private static final String TIME_STAMP_FORMAT_ISO8601 = "yyyy-MM-dd'T'HH:mm:ssZ";
 
+    public static final int TOOL_BAR_ONLY_INDEX = 0;
```

Comment:
Better with an enum.

Reply:
I need it to be an int to compare it with the index 
```java
if (brkPanelViewSelect.getSelectedIndex() == TOOL_BAR_ONLY_INDEX) {
     options.getParamSet(BreakpointsParam.class).setShowIgnoreFilesButtons(false);
}
```
Should I change it and add an extra dataStructure to make the translation from enum to int? I think it would make it worst.

Reply:
The enum values can have the corresponding index.

Reply:
Thx, I believe that now I did what you were asking.